### PR TITLE
Update telepresence quick start

### PR DESCRIPTION
**Pull request by leocontrerasn**
We update the quick start content for telepresence.io

Preview url: 

https://deploy-preview-176--telepresence.netlify.app/docs/latest/quick-start/

Signed-off-by: Leonel Contreras <lcontreras@datawire.io>

### CODE REVIEW

**Comment by leocontrerasn** [comment]
@njayp, @anafilipadealmeida I just did a change in the  md about telepresence commands  (code component), because it was show me an error about unknown language, so I just delete the console prefix to be similar to the qs-go.md (line 132) in the quick start folder, so feel free to update as you need.

File: docs/v2.5/quick-start/index.md [hunk 1]
```diff
@@ -1,7 +1,186 @@
...
+sudo chmod a+x /usr/local/bin/telepresence
+# Apple silicon Macs
+# Install via brew:
+brew install datawire/blackbird/telepresence-arm64
+# OR Install manually:
+# 1. Download the latest binary (~60 MB):
+sudo curl -fL https://app.getambassador.io/download/tel2/darwin/arm64/$dlVersion$/telepresence -o /usr/local/bin/telepresence
+# 2. Make the binary executable:
+sudo chmod a+x /usr/local/bin/telepresence
+```
```
Comment (njayp) [inline docs/v2.5/quick-start/index.md:45]: this is 4 different scripts, not one script, they should not all be copied by the same button

File: docs/v2.5/quick-start/index.md [hunk 2]
```diff
@@ -1,7 +1,186 @@
...
-<TelepresenceQuickStartLanding/>
+# Intercept a service in your own environment
+
+Telepresence enables you to create intercepts to a target Kubernetes workload. Once you have created and intercept, you can code and debug your associated service locally.
+
+For a detailed walk-though on creating intercepts using our sample app, follow the [quick start guide](../../quick-start/demo-node/).
+
+## Prerequisites
+
+Before you begin, you need to have either the Kubernetes command-line tool, [`kubectl`](https://kubernetes.io/docs/tasks/tools/install-kubectl/), or the OpenShift Container Platform command-line interface, [`oc`](https://docs.openshift.com/container-platform/4.2/cli_reference/openshift_cli/getting-started-cli.html#cli-installing-cli_cli-developer-commands). This document uses kubectl in all example commands. OpenShift users can substitute oc [commands instead](https://docs.openshift.com/container-platform/4.1/cli_reference/developer-cli-commands.html).
```
Comment (njayp) [inline docs/v2.5/quick-start/index.md:17]: these links do not look clickable, can you make them blue?

File: docs/v2.5/quick-start/index.md [hunk 3]
```diff
@@ -1,7 +1,186 @@
...
+
+## Install telepresence in your cluster
+
+1. Install the traffic manager into your cluster with `telepresence helm install`. More information about installing telepresence can be found [here](../install/manager/). This will require root access on your machine.
+
+```
+$ telepresence helm install
+  ...
+  Traffic Manager installed successfully
+```
```
Comment (njayp) [inline docs/v2.5/quick-start/index.md:90]: not properly indented

File: docs/v2.5/quick-start/index.md [hunk 4]
```diff
@@ -1,7 +1,186 @@
...
+import Alert from '@material-ui/lab/Alert';
+import Platform from '@src/components/Platform';
+import QSCards from '../quick-start/qs-cards'
 
-<TelepresenceQuickStartLanding/>
+# Intercept a service in your own environment
+
+Telepresence enables you to create intercepts to a target Kubernetes workload. Once you have created and intercept, you can code and debug your associated service locally.
+
+For a detailed walk-though on creating intercepts using our sample app, follow the [quick start guide](../../quick-start/demo-node/).
```
Comment (njayp) [inline docs/v2.5/quick-start/index.md:13]: broken link

**Review by njayp — CHANGES_REQUESTED** [review]

File: docs/v2.5/quick-start/index.md (hunk 1 above)
Comment (leocontrerasn) [reply docs/v2.5/quick-start/index.md:45]: Do you have an example for this? I found this one, but copy all the text: 

Screen Shot 2022-10-13 at 9 53 08

Comment (njayp) [reply docs/v2.5/quick-start/index.md:45]: i dont have an example. maybe make four different shell boxes?

**Review by aaronUnosquare — APPROVED** [review]

**Pull request merged**